install.py

```diff
@@ -52,6 +52,8 @@ def driver(thread_count):
     os.mkdir(fftw_build)
     subprocess.check_call([os.path.join(fftw_src, 'configure'), '--enable-openmp', '--disable-mpi', '--enable-shared', '--with-pic', '--prefix=%s' % fftw_install], cwd=fftw_build)
     subprocess.check_call(['make', 'install', '-j%s' % thread_count], cwd=fftw_build)
+    subprocess.check_call([os.path.join(fftw_src, 'configure'), '--enable-openmp', '--disable-mpi', '--enable-shared', '--with-pic', '--enable-single', '--prefix=%s' % fftw_install], cwd=fftw_build)
+    subprocess.check_call(['make', 'install', '-j%s' % thread_count], cwd=fftw_build)
```

Comment:
I would be more comfortable if we create an entirely separate build directory for the single precision build. Who knows what state might creep over if we reuse the old one.

Reply:
Added separate build directory: https://github.com/arjunkunna/regent-fft/commit/52fb1edb4b89819658b531ee88847918bef62fc8

added this